ui/src/components/song/SongCard.tsx

```diff
@@ -78,35 +99,41 @@ const SongCard = (props: SongCardProps) => {
           {filterData?.display?.lyricsPreview === true ||
           filterData?.display?.lyricsPreview === undefined ? (
             <>
-              <Box
-                aria-owns={open ? 'mouse-over-popover' : undefined}
-                aria-haspopup="true"
-                onMouseEnter={handlePopoverOpen}
-                onMouseLeave={handlePopoverClose}
-                sx={{ height: '30px', width: '30px' }}
-              >
-                <VisibilityIcon sx={{ color: 'primary.main' }} />
+              <Box sx={{ height: '30px', width: '30px' }}>
+                <VisibilityIcon onClick={handleOpen} sx={{ color: 'primary.main' }} />
               </Box>
-              <Popover
-                id="mouse-over-popover"
-                sx={{
-                  pointerEvents: 'none',
-                }}
-                open={open}
-                anchorEl={anchorEl}
-                anchorOrigin={{
-                  vertical: 'bottom',
-                  horizontal: 'left',
-                }}
-                transformOrigin={{
-                  vertical: 'top',
-                  horizontal: 'left',
-                }}
-                onClose={handlePopoverClose}
-                disableRestoreFocus
-              >
-                <Typography sx={{ p: 1 }}>{lyricsPreview}</Typography>
-              </Popover>
+              <Modal open={modalOpen} onClose={handleClose}>
+                <Stack
+                  sx={{
+                    position: 'absolute' as 'absolute',
```

Comment:
why like this ya? `absolute as absolute`

Reply:
nvm i was followin ts position syntax but i realized its useless anyway